src/internal/connector/graph/service.go

```diff
@@ -81,9 +72,17 @@ type Displayable interface {
 	GetDisplayName() *string
 }
 
+// Additionalable represents objects that implement msgraph-sdk-go/models.entityable
+// and have the concept of an additional data map.
+type Additionalable interface {
+	Idable
+	GetAdditionalData() map[string]any
+}
+
```

Comment:
I don't think this is required anymore since we don't pull delta info from the container resolver
```suggestion
```